Code/Framework/AzFramework/AzFramework/Input/Channels/InputChannelId.h

```diff
@@ -40,20 +41,32 @@ namespace AzFramework
         ////////////////////////////////////////////////////////////////////////////////////////////
         //! Constructor
         //! \param[in] name Name of the input channel (will be truncated if exceeds MAX_NAME_LENGTH)
-        explicit InputChannelId(const char* name = "");
-
-        ////////////////////////////////////////////////////////////////////////////////////////////
-        //! Copy constructor
-        //! \param[in] other Another instance of the class to copy from
-        InputChannelId(const InputChannelId& other);
-
-        ////////////////////////////////////////////////////////////////////////////////////////////
-        //! Copy assignment operator
-        //! \param[in] other Another instance of the class to copy from
-        InputChannelId& operator=(const InputChannelId& other);
+        explicit constexpr InputChannelId(const char* name = "")
```

Comment:
This should be changed to accept an `AZStd::string_view`
```suggestion
        explicit constexpr InputChannelId(AZStd::string_view name = "")
```